### DISCUSSION TITLE
No Connection After Updating Hosts File

### DISCUSSION THREAD

**Discussion by MrSome0ne** [StevenBlack/hosts#2107] [Q&A] [answered]
I used a hosts file with unified hosts+social+gambling. Now my laptop is showing no internet

**Comment by StevenBlack** [comment, marked as answer]
Hello @MrSome0ne thank you for your question.

Please see [Warning: Using this hosts file in Windows may require disabling DNS Cache service](https://github.com/StevenBlack/hosts#warning-using-this-hosts-file-in-windows-may-require-disabling-dns-cache-service) section in the readme.

I hope this helps!

**Reply by skoenemann** [reply]
Also, this discussion thread... https://github.com/StevenBlack/hosts/discussions/2029
